### PR TITLE
SERVICES-492 - Added profiler for UserPreferences get/set methods.

### PR DESCRIPTION
**Pull request by harnash**
Added additional (sampled at 1%) log to get/set user preferences.

### CODE REVIEW

**Comment by harnash** [comment]
@Wikia/services-team

**Comment by nmonterroso** [comment]
I thought that there was already a mechanism in MediaWiki for sending timing stats to grafana? /cc @wladekb

**Comment by harnash** [comment]
After consultations with @wladekb I've decided to use Transactions to send data "directly" to InfluxDB. This will allow us to skip writing specific collector which will send data to InfluxDB.

File: lib/Wikia/src/Service/User/PreferenceKeyValueService.php [hunk 1]
```diff
@@ -24,8 +24,13 @@
 use Wikia\Persistence\User\PreferencePersistence;
 
 class PreferenceKeyValueService implements PreferenceService {
+	use \Wikia\Util\WikiaProfiler;
 
+	/**
+	 * @var PreferencePersistence
+     */
```
Comment (wladekb) [inline lib/Wikia/src/Service/User/PreferenceKeyValueService.php]: indentation is off here

**Comment by wladekb** [comment]
lgtm

File: lib/Wikia/src/Service/User/PreferenceKeyValueService.php (hunk 1 above)
Comment (harnash) [inline lib/Wikia/src/Service/User/PreferenceKeyValueService.php]: Fixed in 7422ad68c0e6e4df45feb2dac813fbd9b214d5e6

**Comment by nmonterroso** [comment]
This just feels like we're recreating `wfProfileIn` and `wfProfileOut`.. is this really how we send stuff to InfluxDB? If so I guess LGTM, but I thought we didn't have to do the "start timing", "end timing" anymore

**Comment by nmonterroso** [comment]
LGTM

**Pull request merged**